slices/openjdk-21-jre-headless.yaml

```diff
@@ -0,0 +1,197 @@
+package: openjdk-21-jre-headless
+
+# OpenJDK slices provide required dependencies for the runtime image.
+# The image itself is built using jlink.
+# The slices can be used to create build image.
+
+essential:
+  - openjdk-21-jre-headless_copyright
+
+slices:
+  core:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+      - zlib1g_libs
+    contents:
+      /etc/java-21-openjdk/jaxp.properties:
+      /etc/java-21-openjdk/jvm-*.cfg:
+      /etc/java-21-openjdk/logging.properties:
+      /etc/java-21-openjdk/net.properties:
+      /etc/java-21-openjdk/security/java.policy:
+      /etc/java-21-openjdk/security/java.security:
+      /usr/lib/jvm/java-21-openjdk-*/bin/java:
+      /usr/lib/jvm/java-21-openjdk-*/conf/jaxp.properties:
+      /usr/lib/jvm/java-21-openjdk-*/conf/logging.properties:
+      /usr/lib/jvm/java-21-openjdk-*/conf/net.properties:
+      /usr/lib/jvm/java-21-openjdk-*/conf/security/java.policy:
+      /usr/lib/jvm/java-21-openjdk-*/conf/security/java.security:
+      /usr/lib/jvm/java-21-openjdk-*/lib/jexec:
+      /usr/lib/jvm/java-21-openjdk-*/lib/jspawnhelper:
+      /usr/lib/jvm/java-21-openjdk-*/lib/jvm.cfg:
+      /usr/lib/jvm/java-21-openjdk-*/lib/jvm.cfg-default:
+      /usr/lib/jvm/java-21-openjdk-*/lib/libextnet.so:
+      /usr/lib/jvm/java-21-openjdk-*/lib/libjava.so:
+      # lib/modules support
+      /usr/lib/jvm/java-21-openjdk-*/lib/libjimage.so:
+      /usr/lib/jvm/java-21-openjdk-*/lib/libjli.so:
+      /usr/lib/jvm/java-21-openjdk-*/lib/libjsig.so:
+      # terminal support
+      /usr/lib/jvm/java-21-openjdk-*/lib/lible.so:
+      /usr/lib/jvm/java-21-openjdk-*/lib/libnet.so:
+      /usr/lib/jvm/java-21-openjdk-*/lib/libnio.so:
```

Comment:
is this really needed in `core`? same for `nio` and the other groupings.

Reply:
console and prefs can be moved out. 
I am not sure about removing net/nio since those are required socket operations.

Reply:
nio is required for IO internally as it provides FileSystem implementation.